content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md

```diff
@@ -348,6 +348,13 @@ Footnotes are not supported in wikis.
 
 **Note**: The Markdown syntax used during the beta testing period is now deprecated and will be removed. You can use the syntax as described in this section.
 
+The deprecated syntax that was previously used is as follow:
```

Comment:
```suggestion
The deprecated syntax is as follows:
```

We can remove "that was previously used" here to avoid repetition with "deprecated."